lms/static/sass/_pearsonx.scss

```diff
@@ -213,9 +213,16 @@ h2 {
         width: 100%;
         height: auto;
       }
+      // Hide the picture for small screens OC-4050
```

Comment:
@xitij2000 👍 for referencing the ticket number here. Let's change the format slightly and provide reasoning for hiding the picture:

```css
// OC-4050: Hide the picture for small screens to make sure viewport centers on contact form.
```